dataclients/kubernetes/eastwest.go

```diff
@@ -43,3 +46,52 @@ func createEastWestRouteRG(name, ns, postfix string, r *eskip.Route) *eskip.Rout
 	ewr.Predicates = p
 	return ewr
 }
+
+func applyEastWestRange(eWRange map[string]string, host string, routes []*eskip.Route) {
+	for d, n := range eWRange {
+		if strings.HasSuffix(host, d) {
+			applyEastWestRangePredicates(routes, n)
+		}
+	}
+}
+
+func applyEastWestRangePredicates(routes []*eskip.Route, n string) {
+	for _, route := range routes {
+		route.Predicates = append(route.Predicates, &eskip.Predicate{
+			Name: "SourceFromLast",
+			Args: []interface{}{n},
+		})
+	}
+}
+
+// ParseEastWestRange parses the multiple strings representations
+// of the internal domains and their CIDRs.
+func ParseEastWestRange(v []string) (map[string]string, error) {
+	eastWestRange := map[string]string{}
+	for _, i := range v {
```

Comment:
i seems to be a bad name. Maybe cidr, but i is normally the index var not a value which is not an int

Reply:
I was operating with the index in the very beggining and forgot to update. Thanks!